<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Form</class>
 <widget class="QWidget" name="Main_Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/iptables_resource.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="accessibleName">
   <string>iptables GUI</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>61</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <attribute name="title">
     <string>INPUT</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>771</width>
       <height>41</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1_1">
      <attribute name="title">
       <string>filter</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_1_2">
      <attribute name="title">
       <string>nat</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_1_3">
      <attribute name="title">
       <string>mangle</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>OUTPUT</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2_1">
      <attribute name="title">
       <string>filter</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2_2">
      <attribute name="title">
       <string>nat</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2_3">
      <attribute name="title">
       <string>mangle</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>FORWARD</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3_1">
      <attribute name="title">
       <string>filter</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3_2">
      <attribute name="title">
       <string>mangle</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>PREROUTING</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4_1">
      <attribute name="title">
       <string>nat</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4_2">
      <attribute name="title">
       <string>mangle</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>POSTROUTING</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5_1">
      <attribute name="title">
       <string>nat</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_5_2">
      <attribute name="title">
       <string>mangle</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>751</width>
     <height>351</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rule</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons/iptables_resource.qrc">
      <normaloff>:/set/set.png</normaloff>:/set/set.png</iconset>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Creator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons/iptables_resource.qrc">
     <normaloff>:/make_img/make.png</normaloff>:/make_img/make.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Rule</string>
   </property>
  </widget>
  <action name="actionmake_image">
   <property name="icon">
    <iconset>
     <normaloff>make.png</normaloff>make.png</iconset>
   </property>
   <property name="text">
    <string>make_image</string>
   </property>
  </action>
  <action name="actiondelete_image">
   <property name="icon">
    <iconset>
     <normaloff>delete.png</normaloff>delete.png</iconset>
   </property>
   <property name="text">
    <string>delete_image</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget_main</tabstop>
  <tabstop>tabWidget_1</tabstop>
 </tabstops>
 <resources>
  <include location="icons/iptables_resource.qrc"/>
 </resources>
 <connections/>
</ui>
